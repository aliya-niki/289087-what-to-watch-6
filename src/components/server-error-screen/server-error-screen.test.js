import React from 'react';
import {render} from '@testing-library/react';
import ServerErrorScreen from './server-error-screen';

test(`ServerErrorScreen should render correctly`, () => {
  const {container} = render(<ServerErrorScreen />);

  expect(container).toMatchSnapshot();
});
