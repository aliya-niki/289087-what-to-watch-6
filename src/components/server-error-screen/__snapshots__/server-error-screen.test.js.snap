// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServerErrorScreen should render correctly 1`] = `
<div>
  <p>
    Sorry, the server is temporarily unavailable
  </p>
</div>
`;
